crowdfunding/crowdfunding/settings.py

```diff
@@ -20,27 +21,46 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-6f5fiv53l$d=%d_0_8&znvd!6&d3rfy-qowzswx^u)i-p_dsm6'
-
+SECRET_KEY = os.environ.get(
+    'DJANGO_SECRET_KEY',
+    'django-insecure-6f5fiv53l$d=%d_0_8&znvd!6&d3rfy-qowzswx^u)i-p_dsm6'
+)
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.environ.get(
+   'DJANGO_DEBUG', 
+   'False'
+) != 'False'
+
+STATIC_ROOT = BASE_DIR / 'staticfiles'
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```

Comment:
this is fine for your proof of concept site, but if you were going to use this as a production server, you'd probably want to have this very specifically only refer to the sites you will access from